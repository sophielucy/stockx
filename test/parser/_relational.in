true;
false;
not true;
not false;

1 == 2;
3 < 4;
5 <= 6;
7 > 0;
1 >= -5;
not (1 < 5);
