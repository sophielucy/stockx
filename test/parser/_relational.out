BoolLiteral(true)
BoolLiteral(false)
Unop(Not, BoolLiteral(true))
Unop(Not, BoolLiteral(false))
Binop(IntLiteral(1), Equal, IntLiteral(2))
Binop(IntLiteral(3), Less, IntLiteral(4))
Binop(IntLiteral(5), Leq, IntLiteral(6))
Binop(IntLiteral(7), Greater, IntLiteral(0))
Binop(IntLiteral(1), Geq, Unop(Neg, IntLiteral(5)))
Unop(Not, Binop(IntLiteral(1), Less, IntLiteral(5)))
